recipe/patches/0006-use-find-python.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/src/python/CMakeLists.txt b/src/python/CMakeLists.txt
+index 337b61aa..208cffad 100644
+--- a/src/python/CMakeLists.txt
++++ b/src/python/CMakeLists.txt
+@@ -1,4 +1,4 @@
+-find_package(Python3 COMPONENTS Interpreter Development.Module REQUIRED)
++find_package(Python COMPONENTS Interpreter Development.Module REQUIRED)
```

Comment:
are things using the `Python_` macros, or `Python3_`

Reply:
Neither .. it's just used as a hint for pybind11. Anyhow, it doesn't solve the problem :(